Amazon.IonDotnet/Internals/Binary/SymbolTableReader.cs

```diff
@@ -28,41 +28,41 @@ namespace Amazon.IonDotnet.Internals.Binary
     /// </summary>
     internal class SymbolTableReader : IIonReader
     {
-        private const int S_BOF = 0;
-        private const int S_STRUCT = 1;
-        private const int S_IN_STRUCT = 2;
-        private const int S_NAME = 3;
-        private const int S_VERSION = 4;
-        private const int S_MAX_ID = 5;
-        private const int S_IMPORT_LIST = 6;
-        private const int S_IN_IMPORTS = 7;
-        private const int S_IMPORT_STRUCT = 8;
-        private const int S_IN_IMPORT_STRUCT = 9;
-        private const int S_IMPORT_NAME = 10;
-        private const int S_IMPORT_VERSION = 11;
-        private const int S_IMPORT_MAX_ID = 12;
-        private const int S_IMPORT_STRUCT_CLOSE = 13;
-        private const int S_IMPORT_LIST_CLOSE = 14;
-        private const int S_AFTER_IMPORT_LIST = 15;
-        private const int S_SYMBOL_LIST = 16;
-        private const int S_IN_SYMBOLS = 17;
-        private const int S_SYMBOL = 18;
-        private const int S_SYMBOL_LIST_CLOSE = 19;
-        private const int S_STRUCT_CLOSE = 20;
-        private const int S_EOF = 21;
-
-        private const int HAS_NAME = 0x01;
-        private const int HAS_VERSION = 0x02;
-        private const int HAS_MAX_ID = 0x04;
-        private const int HAS_IMPORT_LIST = 0x08;
-        private const int HAS_SYMBOL_LIST = 0x10;
+        private const int SBOF = 0;
```

Comment:
This change makes these constants less readable. Is the problem that StyleCop doesn't like the underscores? What style does it recommend for constants? Upper camel case? If so, let's use that, as it will be more readable than all caps.

Reply:
These are the recommended changed names by the IDE.

Let me revisit this to see if there's a format that's both more readable and adheres to C# best practices.

Reply:
Renamed the fields to be more readable